npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater.rb

```diff
@@ -574,18 +537,6 @@ def yarnrc_content
           ).yarnrc_content
         end
 
-        def sanitized_package_json_content(content)
-          updated_content =
-            content.
-            gsub(/\{\{[^\}]*?\}\}/, "something"). # {{ nm }} syntax not allowed
-            gsub(/(?<!\\)\\ /, " ").          # escaped whitespace not allowed
-            gsub(%r{^\s*//.*}, " ")           # comments are not allowed
-
-          json = JSON.parse(updated_content)
-          json["name"] = json["name"].delete(" ") if json["name"].is_a?(String)
```

Comment:
This cleanup is no longer done after this refactoring. I think it's no longer needed but I'll extract its removal to a separate PR.

Reply:
This was extracted into https://github.com/dependabot/dependabot-core/pull/7246 and deployed a couple of weeks ago.